09_maps/main.go

```diff
@@ -69,22 +69,39 @@ func main() {
 	fmt.Println(human)
 
 	// TODO: Print the length of the map human
+	fmt.Println(len(human))
 
 	// TODO: Check if the key "age" exists in the map human
 	// If so print "age exists in the map"
 	// If not print "age does not exist in the map"
 	// Do the same for the key "weight"
+	var value int
+	var exists bool
+	value, exists = human["age"]
+	if exists {
+		fmt.Println("age exists in the map")
+	}
+	if value == 0 {
+		fmt.Println("age does not exist in the map")
+	}
```

Comment:
This is incorrect, `if value == 0` means that the value is equal to `0`, not that it doesn't exist in the map. Using the variable `exists` that you defined above is the only way to check if the value exists.

Reply:
If/if (or if/else) statement doesn't run, currently get`declared and not used: value`, but seem to need 2 variables to be able to check correctly.
 
`To check if a key exists in a map, you can use the following syntax:`
seems to be missing in the instructions.
The next mentioned syntax to get the value seemed to work, except it doesn't :(

Reply:
> seems to be missing in the instructions.

Oh yeah, it's badly written. It should have been:

```
To check if a key exists in a map and to get the value, you can use the following syntax:

    var value int
    var exists bool
    value, exists = m["age"]

```

Reply:
Printed it. What if I don't want to have it printed?

Reply:
There's a way to discard values in go, if you want to read about it: https://go.dev/doc/effective_go#blank